vmdk_plugin/plugin.go

```diff
@@ -249,10 +249,10 @@ func (d *vmdkDriver) Create(r volume.Request) volume.Response {
 	if err != nil {
 		log.WithFields(log.Fields{"name": r.Name,
 			"error": err}).Error("Attach volume failed, removing the volume ")
-		err = d.ops.Remove(r.Name, nil)
-		if err != nil {
-			log.WithFields(log.Fields{"name": r.Name, "error": err}).Error("Remove volume failed ")
-			return volume.Response{Err: err.Error()}
+		errRm := d.ops.Remove(r.Name, nil)
+		if errRm != nil {
+			log.WithFields(log.Fields{"name": r.Name, "error": errRm}).Error("Remove volume failed ")
+			return volume.Response{Err: errRm.Error()}
```

Comment:
Its preferable that the error returned on a create error is the create error itself vs. a remove error (which is confusing to the user). OTOH, how did this cause the empty response from docker to the docker command line.